plugins/janus_videoroom.c

```diff
@@ -785,6 +785,19 @@ room-<unique room ID>: {
 		// Other publishers
 	]
 }
+\endverbatim *
+ *
+ * To enable or disable recording while the conference is in progress,
+ * you can make use of the \c enable_recording request, which has to be
+ * formatted as follows:
+ *
+\verbatim
+{
+	"request" : "enable_recording",
+	"room" : <unique numeric ID of the room>,
+	"secret" : "<room secret; mandatory if configured>"
+	"record" : <true|false, whether this room should be recorded or not>,
```

Comment:
I'd say "whether participants in this room should be automatically recorded or not".